src/main/resources/application.yml

```diff
@@ -21,7 +21,7 @@ dbcollections:
       
 server: 
     port: 8085
-    contextPath: /oar-rmm-service
+    contextPath: /rmm
```

Comment:
@deoyani, @RayPlante  please make sure this change is also reflected in the nginx config since currently it is pointing at oar-rmm-service